homeassistant/components/humidifier/intent.py

```diff
@@ -41,8 +41,14 @@ async def async_handle(self, intent_obj: intent.Intent) -> intent.IntentResponse
         """Handle the hass intent."""
         hass = intent_obj.hass
         slots = self.async_validate_slots(intent_obj.slots)
-        state = intent.async_match_state(
-            hass, slots["name"]["value"], hass.states.async_all(DOMAIN)
+        state = next(
+            iter(
+                intent.async_match_states(
+                    hass,
+                    name=slots["name"]["value"],
+                    states=hass.states.async_all(DOMAIN),
+                )
+            )
```

Comment:
You want to add a default to `Next` or else you end up raising `StopIteration` and no one is any wiser.
```suggestion
            ), None
```

Reply:
Added a property check and error.